packages/account-v2/src/constants/manualDocument.tsx

```diff
@@ -23,7 +23,7 @@ export const getManualUploadDocumentList = (isNimcRequired: boolean): TDocumentT
             description: 'Upload the front and back of your driving licence.',
             icon: IcPoiDrivingLicence,
             title: 'Driving licence',
-            value: 'driving_licence',
+            value: 'driving_license',
```

Comment:
@suisin-deriv, can u recheck on this value. I think BE uses `driving_licence`. Though both spelling are right